diff --git a/ucm2/Qualcomm/sc8280xp/HiFi.conf b/ucm2/Qualcomm/sc8280xp/HiFi.conf
index 0fb5302..2dd3220 100644
--- a/ucm2/Qualcomm/sc8280xp/HiFi.conf
+++ b/ucm2/Qualcomm/sc8280xp/HiFi.conf
@@ -27,7 +27,7 @@ SectionDevice."Speaker" {
 
 	Value {
 		PlaybackPriority 100
-		PlaybackPCM "hw:${CardId},1"
+		PlaybackPCM "hw:${CardId},5"
 		PlaybackMixer "default:${CardId}"
 		PlaybackMixerElem "Speakers"
 	}
@@ -43,7 +43,7 @@ SectionDevice."Headphones" {
 
 	Value {
 		PlaybackPriority 200
-		PlaybackPCM "hw:${CardId},0"
+		PlaybackPCM "hw:${CardId},4"
 		PlaybackMixer "default:${CardId}"
 		PlaybackMixerElem "HP"
 		JackControl "Headphone Jack"
@@ -61,7 +61,7 @@ SectionDevice."Mic" {
 
 	Value {
 		CapturePriority 100
-		CapturePCM "hw:${CardId},2"
+		CapturePCM "hw:${CardId},6"
 		CaptureMixerElem "ADC2"
 		JackControl "Mic Jack"
 		JackHWMute "DMic01"
@@ -79,6 +79,6 @@ SectionDevice."DMic01" {
 
 	Value {
 		CapturePriority 100
-		CapturePCM "hw:${CardId},3"
+		CapturePCM "hw:${CardId},7"
 	}
 }
